base/docs/helpdb/Base.jl

```diff
@@ -130,13 +130,28 @@ Obtain the contents of an `IOBuffer` as an array, without copying. Afterwards, t
 takebuf_array
 
 """
-    download(url,[localfile])
+    download(url [localfile] [silent=false])
```

Comment:
I was having difficulty getting doc generation to work and trying to figure out why it was not matching the doc signatures. Editing the .rst files worked,I should not have pushed those changes. My current push corrects that.